e2e/libs/backup/rest.py

```
@@ -157,9 +160,16 @@ def delete(self, volume_name, backup_id):
-        bv = get_longhorn_client().by_id_backupVolume(volume_name)
-        get_longhorn_client().delete(bv)
-        self.wait_for_backup_volume_delete(volume_name)
+        bvs = get_longhorn_client().list_backupVolume()
+        backup_volume_name = None
+        for bv in bvs:
+            volumeName = getattr(bv, 'volumeName', bv.name)
+            if volumeName == volume_name:
+                backup_volume_name = bv.name
+                get_longhorn_client().delete(bv)
+                break
+        if backup_volume_name != None:
```

Ruff (0.8.2)

Comparison to `None` should be `cond is not None`

Replace with `cond is not None`

(E711)